src/sentry/static/sentry/app/utils/discover/fields.tsx

```diff
@@ -485,13 +485,19 @@ export enum WebVital {
   FCP = 'measurements.fcp',
   LCP = 'measurements.lcp',
   FID = 'measurements.fid',
+  CLS = 'measurements.cls',
+  TTFB = 'measurements.ttfb',
+  RequestTime = 'measurements.ttfb.requesttime',
 }
 
 const MEASUREMENTS: Readonly<Record<WebVital, ColumnType>> = {
   [WebVital.FP]: 'duration',
   [WebVital.FCP]: 'duration',
   [WebVital.LCP]: 'duration',
   [WebVital.FID]: 'duration',
+  [WebVital.CLS]: 'number',
+  [WebVital.TTFB]: 'duration',
+  [WebVital.RequestTime]: 'duration',
 };
```

Comment:
Should this new data be added to the `WEB_VITALS_DETAILS`? Keeping the details data and these enums aligned feels like we are making our lives hard.

Reply:
Currently, `WEB_VITAL_DETAILS` reads in these values via `measurementType`, but I think you're right. To simplify this, we can have all the type information in `WEB_VITAL_DETAILS` and have `measurementType` read in the type information from there instead.

Reply:
It's already there. This is used as a source of truth and added to `WEB_VITALS_DETAILS`